core/css.js

```diff
@@ -192,7 +192,6 @@ Blockly.Css.CONTENT = [
     'border: 1px solid;',
     'border-radius: 2px;',
     'padding: 4px;',
-    '-webkit-user-select: none;',
```

Comment:
This line is the only functional change.  First, it's broken, only applying to webkit.  Second, it is inappropriate for the dropdowndiv to declare that any contents be unselectable; we have no idea what's in dropdowndiv.  Currently all content in existing dropdowndivs is non-text, with the exception of the Turtle demo.  I've added blocklyNonSelectable to that demo.